src/workloads/scale/MemoryStress.yml

```diff
@@ -0,0 +1,90 @@
+SchemaVersion: 2018-07-01
+Owner: Service Architecture
+Description: |
+  Used to test performance of the server under heavy memory pressure.
```

Comment:
As an effect of the purpose described here, the memory heavy operations should cause the hosts associated with the latest mongod binaries to crash. This is because we currently do not have anything to prevent memory overload.

The ideal purpose for putting this crash inducing workload into master is so that other server engineers can find this as an Evergreen task and run to test their implementations against. But because this would cause red/purple-ness on sys-perf with the current mongod binaries, I want to leave this whole workload commented out. Future engineers who want to use this workload would simply uncomment and run `schedule_patch_auto_tasks`.

In the future, I expect this to be uncommented as mongod gets better at handling memory pressure (and therefore would no longer cause failures).